cli/src/main/scala/org/apache/celeborn/cli/master/MasterSubcommandImpl.scala

```diff
@@ -224,18 +225,39 @@ class MasterSubcommandImpl extends Runnable with MasterSubcommand {
   private[master] def runShowContainerInfo: ContainerInfo = defaultApi.getContainerInfo
 
   override private[master] def reviseLostShuffles: HandleResponse = {
+    if (StringUtils.isEmpty(commonOptions.apps)) {
```

Comment:
Could this condition combine with line 240?

Reply:
sure

Reply:
thanks, done